tests/integration/test_charm_one_runner.py

```diff
@@ -291,3 +296,40 @@ async def test_token_config_changed_insufficient_perms(
     await model.wait_for_idle()
 
     await wait_till_num_of_runners(unit, num=0)
+
+
+@pytest.mark.parametrize(
+    "image",
+    [
+        pytest.param("noble", id="noble"),
```

Comment:
should we leave this as not parametrised until we need o test for more ubuntu OS?